modules/tactical/src/idb.ts

```diff
@@ -18,15 +19,23 @@ export interface Idb {
  * An in-memory implementation of the Idb interface, mostly useful for testing.
  */
 export class InMemoryIdb implements Idb {
-  db: Object = {};
+  db: Object;
+
+  /**
+   * Opens a connection to the Idb database.
+   */
+  open(database: string, stores: string[]) {
+    this.db = {};
+    stores.forEach((store: string) => { this.db[store] = {}; });
+  }
```

Comment:
So, does this mean that only one database is possible.
Trying to access stores in different databases will silently return the same instance (as long as the store names match).

Is this intentional ?